app/controllers/YourAccountController.scala

```diff
@@ -58,6 +59,8 @@ class YourAccountController @Inject()(
                                      )(implicit ec: ExecutionContext)
   extends FrontendBaseController with I18nSupport with Logging {
 
+  implicit val c: Clock = clock
```

Comment:
The clock could be moved to the list of implicits: `(implicit ec: ExecutionContext, clock: Clock)`